packages/backend/scripts/configure-env.ts

```diff
@@ -0,0 +1,81 @@
+// import fs from 'fs';
```

Comment:
Is this needed

Reply:
There are a lot more commented out lines of code but forgot to include in review

Reply:
Also it seems like `packages/backend/scripts/configure-env.ts` and `packages/backend/src/scripts/configure-env.ts` are meant to do the same thing except `packages/backend/scripts/configure-env.ts` is missing the parser